### ISSUE TITLE
Refactor REACT JS code

### ISSUE THREAD

**Issue by bmorelli25** [bmorelli25/ReactWeather#46]
Hello [Hacktoberfest](https://hacktoberfest.digitalocean.com/)!

Are you interested in React JS, Node JS, APIs, HTML, Javascript, Webpack, Express, and/or SASS/CSS? Then you've come to the right project!

This web app allows a user to type in a location, then pulls weather data from [OpenWeatherMap's API](http://openweathermap.org/) and returns it to the user. **[Live Demo](http://still-springs-11020.herokuapp.com/)**

**If this issue doesn't interest you, but there are other issues, bugs, enhancements you would like to make, please feel free to open up a new Issue and we can get a discussion started. I want everyone to be able to contribute!**

Your task: Refactor `WeatherForecastList.jsx`. Currently, the component returns a map of `WeatherForecastDay.jsx` modules:

```
return newData.map((day) => {
                day.date_text = moment.unix(day.dt).format('dddd, MMM D');
                return <WeatherForecastDay tempType={tempType} key={day.dt} {...day}/>
            });
```

This works great. However, we also want the following to be returned (just once, at the top):

```
<h4 className="forecast-title">Forecast for next 5 days</h4>
```

Don't hesitate to reach out with questions, or if you need help understanding the code!

_PS: This project utilizes a `.editorconfig` file. This helps developers define and maintain consistent coding styles between different editors and IDEs. Please ensure your editor is set up to use the `.editorconfig` file. For more information, check out [editorconfig.org](http://editorconfig.org/)._

**Comment by mmhansen** [comment]
I can handle this one if no one else has taken it.

**Comment by bmorelli25** [comment]
hey @mmhansen - sorry for the delay on getting back to you. This was fixed with #54. Please feel free to open a new issue if there is anything you would like to add.
